src/main/java/uk/gov/hmcts/reform/sscs/callback/handlers/ManualCaseCreatedHandler.java

```diff
@@ -1,61 +1,54 @@
 package uk.gov.hmcts.reform.sscs.callback.handlers;
 
+import static java.util.Collections.singletonMap;
 import static java.util.Objects.requireNonNull;
+import static uk.gov.hmcts.reform.sscs.callback.handlers.helper.WorkAllocationFieldsHelper.setWorkAllocationFields;
+import static uk.gov.hmcts.reform.sscs.ccd.callback.CallbackType.SUBMITTED;
+import static uk.gov.hmcts.reform.sscs.ccd.callback.DispatchPriority.LATEST;
+import static uk.gov.hmcts.reform.sscs.ccd.domain.EventType.INCOMPLETE_APPLICATION_RECEIVED;
+import static uk.gov.hmcts.reform.sscs.ccd.domain.EventType.NON_COMPLIANT;
+import static uk.gov.hmcts.reform.sscs.ccd.domain.EventType.VALID_APPEAL_CREATED;
 
 import java.util.HashMap;
 import java.util.Map;
+import lombok.RequiredArgsConstructor;
 import lombok.extern.slf4j.Slf4j;
-import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Service;
 import uk.gov.hmcts.reform.sscs.callback.CallbackHandler;
 import uk.gov.hmcts.reform.sscs.ccd.callback.Callback;
 import uk.gov.hmcts.reform.sscs.ccd.callback.CallbackType;
 import uk.gov.hmcts.reform.sscs.ccd.callback.DispatchPriority;
-import uk.gov.hmcts.reform.sscs.ccd.domain.*;
+import uk.gov.hmcts.reform.sscs.ccd.domain.SscsCaseData;
 import uk.gov.hmcts.reform.sscs.ccd.service.CcdService;
 import uk.gov.hmcts.reform.sscs.idam.IdamService;
 import uk.gov.hmcts.reform.sscs.idam.IdamTokens;
 
 @Slf4j
 @Service
+@RequiredArgsConstructor
 public class ManualCaseCreatedHandler implements CallbackHandler<SscsCaseData> {
 
-    private final DispatchPriority dispatchPriority;
-
     private final CcdService ccdService;
-
     private final IdamService idamService;
 
-    private Map<String, Object> hmctsServiceIdMap;
-
-    private Map<String, Map<String, Object>> supplementaryDataRequestMap;
-
-    @Autowired
-    public ManualCaseCreatedHandler(CcdService ccdService,
-                                 IdamService idamService) {
-        this.dispatchPriority = DispatchPriority.LATEST;
-        this.ccdService = ccdService;
-        this.idamService = idamService;
-        hmctsServiceIdMap = new HashMap<>();
-        hmctsServiceIdMap.put("HMCTSServiceId", "BBA3");
-        supplementaryDataRequestMap = new HashMap<>();
-        supplementaryDataRequestMap.put("$set", hmctsServiceIdMap);
-    }
+    @Value("${feature.work-allocation.enabled}")
+    private boolean workAllocationFeature;
```

Comment:
Is this the feature flag we are trying to use for this (case access management) release ?

Reply:
Potentially yes, can be changed if needed.

Reply:
Please do. We will have a work allocation feature flag as well and we cannot confuse the two.